nornir_napalm/plugins/tasks/napalm_configure.py

```diff
@@ -38,9 +44,9 @@ def napalm_configure(
     dry_run = task.is_dry_run(dry_run)
     if not dry_run and diff:
         if commit_message:
-            device.commit_config(message=commit_message)
+            device.commit_config(message=commit_message, revert_in=revert_in)
         else:
-            device.commit_config()
+            device.commit_config(revert_in=revert_in)
```

Comment:
We should just add some logic here to only use the `revert_in` argument if it not None. This is probably less risky for any case where someone is using a NAPALM driver/community driver in some way that has not implemented `revert_in`.